seed/challenges/05-apis-and-microservices/basic-node-and-express.json

```diff
@@ -128,14 +128,14 @@
       "id": "587d7fb1367417b2b2512bf1",
       "title": "Serve JSON on a specific route",
       "description": [
-        "While an HTML server serves (you guessed it ?) HTML, an API serves data. A REST API allows data exchange in a simple way, without the need for clients to know any detail about the server. The client only needs to know where the resource is (the URL), and the action it wants to perform on it (the verb). The GET verb is used when you are fetching some information, without modifying anything. In these days the preferred data format to move these informations around the web is JSON. In few words JSON is a convenient way to represent a JavaScript object as a string, so it can be easily transmitted.",
-        "Let’s create a route that responds with JSON at the path (/json). You can do it as usual, with the app.get() method. Inside the route handler use the method res.json(<object>). This method closes the request-response loop, returning the data. Behind the scenes it converts a valid Javascript object into a string. Then sets the appropriate headers to tell your browser that you are serving JSON, and sends the data back. A valid object has the usual structure {key: data}. data can ba a number, a string, a nested objects or an array. data can also be a variable or the result of a function call. It will be evaluated before converting into a string.",
-        "Serve the object <code>{message: 'json'}</code> as a response in JSON format, to the GET requests to the route /json. Point your browser to <your-app-url>/json, you should see the message on the screen."
+        "While an HTML server serves (you guessed it!) HTML, an API serves data. A REST (representational state transfer) API allows data exchange in a simple way, without the need for clients to know any detail about the server. The client only needs to know where the resource is (the URL), and the action it wants to perform on it (the verb). The GET verb is used when you are fetching some information, without modifying anything. These days, the preferred data format for moving information around the web is JSON. Simply put, JSON is a convenient way to represent a JavaScript object as a string, so it can be easily transmitted.",
```

Comment:
In this section, yes. So I can def do that, however, until #13164 is merged (but this and #13161 should be merged first still), wrapping it in `<dfn>` will only italicize it, since the backend challenge component does not have the `.challenge-instructions` class yet (see #13164)